patroni/__init__.py

```diff
@@ -8,12 +15,41 @@
 
 
 def fatal(string: str, *args: Any) -> None:
+    """Write a fatal message to stderr and exit with code ``1``.
+
+    :param string: message to be written before exiting.
+    """
     sys.stderr.write('FATAL: ' + string.format(*args) + '\n')
     sys.exit(1)
```

Comment:
Since we deprecated python 2.7 it could be changed to:
```suggestion
    sys.exit('FATAL: ' + string.format(*args))
```